README.md

```diff
@@ -114,5 +115,20 @@ Set `$GOPATH` variable to point to your Go workspace directory and add `$GOPATH/
 6. Run the server and the signer using
 
   ```sh
-  goreman start
+  docker-compose up -d
   ```
+
+7. Provision a log and a map 
+```sh
+go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=LOG
```

Comment:
It worked for me (I had to change from localhost to the IP assigned to the container by the docker engine).